sunpy/map/tests/test_mapbase.py

```diff
@@ -365,7 +365,7 @@ def test_resample_metadata(generic_map, sample_method, new_dimensions):
     assert resampled_map.meta['crval2'] == generic_map.center.y.value
     for key in generic_map.meta:
         if key not in ('cdelt1', 'cdelt2', 'crpix1', 'crpix2',
-                       'crval1', 'crval2'):
+                       'crval1', 'crval2', 'naxis1', 'naxis2'):
```

Comment:
Actually, I had changed these things in my forked repo and I didn't delete the previous changes. As a result, when I am opened a PR for this issue, all the previous changes are being shown with it.